testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/KeycloakRealmImportOperatorProvisionerTest.java

```diff
@@ -0,0 +1,373 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.keycloak.k8s.v2alpha1.Keycloak;
+import org.keycloak.k8s.v2alpha1.KeycloakRealmImport;
+import org.keycloak.k8s.v2alpha1.KeycloakRealmImportSpec;
+import org.keycloak.k8s.v2alpha1.KeycloakSpec;
+import org.keycloak.k8s.v2alpha1.keycloakrealmimportspec.Realm;
+import org.keycloak.k8s.v2alpha1.keycloakrealmimportspec.realm.Users;
+import org.keycloak.k8s.v2alpha1.keycloakrealmimportspec.realm.users.Credentials;
+import org.keycloak.k8s.v2alpha1.keycloakspec.Db;
+import org.keycloak.k8s.v2alpha1.keycloakspec.Hostname;
+import org.keycloak.k8s.v2alpha1.keycloakspec.Http;
+import org.keycloak.k8s.v2alpha1.keycloakspec.Ingress;
+import org.keycloak.k8s.v2alpha1.keycloakspec.db.PasswordSecret;
+import org.keycloak.k8s.v2alpha1.keycloakspec.db.UsernameSecret;
+import org.slf4j.event.Level;
+
+import cz.xtf.core.openshift.OpenShiftWaiters;
+import cz.xtf.core.openshift.OpenShifts;
+import cz.xtf.core.waiting.SimpleWaiter;
+import cz.xtf.junit5.annotations.CleanBeforeAll;
+import io.fabric8.kubernetes.api.model.DeletionPropagation;
+import io.fabric8.kubernetes.api.model.KubernetesResourceList;
+import io.fabric8.kubernetes.client.dsl.NonNamespaceOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Test the Keycloak Operator provisioning model and APIs
+ *
+ * See <br>
+ *     - https://github.com/keycloak/keycloak-operator/tree/master/deploy/examples
+ * <p>
+ * Not all examples are actually run, but these tests are more about the framework functionality verification than
+ * about the Keycloak operator testing.
+ */
+@Slf4j
+@CleanBeforeAll
+//@Disabled("WIP - Disabled until global-test.properties is configured with the required property")
+public class KeycloakRealmImportOperatorProvisionerTest {
+	private static KeycloakRealmImportOperatorProvisioner KEYCLOAK_OPERATOR_PROVISIONER;
+
+	private static final String POSTGRESQL_NAME = "postgresql";
+	private static final String POSTGRESQL_DATABASE = "keycloak";
+	private static final String POSTGRESQL_PASSWORD = "pippobaudo1234";
+	private static final String POSTGRESQL_USER = "user09M";
+
+	private static final PostgreSQLImageOpenShiftApplication pgSQLApplication = new PostgreSQLImageOpenShiftApplication() {
+		@Override
+		public String getName() {
+			return POSTGRESQL_NAME;
+		}
+
+		@Override
+		public String getUser() {
+			return POSTGRESQL_USER;
+		}
+
+		@Override
+		public String getPassword() {
+			return POSTGRESQL_PASSWORD;
+		}
+
+		@Override
+		public String getDbName() {
+			return POSTGRESQL_DATABASE;
+		}
+	};
+	private static final PostgreSQLImageOpenShiftProvisioner POSTGRESQL_IMAGE_PROVISIONER = new PostgreSQLImageOpenShiftProvisioner(
+			pgSQLApplication);
+
+	private static KeycloakRealmImportOperatorProvisioner initializeOperatorProvisioner(final Keycloak keycloak,
```

Comment:
As said, remove parameters and use dummy values for configuring this application descriptor, it won't be used anyway, we don't deploy via the provisioner here, but use its APIs to create and validate CRs in test methods, see the [InfinispanOperatorTest](https://github.com/Intersmash/intersmash/blob/main/testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/InfinispanOperatorProvisionerTest.java#L66) for inspiration.

Reply:
I agree, but the values in that resource are used in the provisioner for other purposes and I prefer to have them aligned to what we are actually using in the specific test;

Reply:
Can you elaborate about which such purposes are?

Reply:
This can be resolved since this way of initializing the provisioner in the test method is legit in this specific case, provided we implement the fixes detaild in previous comments. Resolving.